console/program/src/request/mod.rs

```diff
@@ -100,6 +117,11 @@ impl<N: Network> Request<N> {
         &self.caller
     }
 
+    /// Returns the request parent.
+    pub const fn parent(&self) -> &Address<N> {
+        &self.parent
```

Comment:
```suggestion
    pub const fn parent(&self) -> Address<N> {
        self.parent
```
the `Address` (the `console` one, not the `circuit` counterpart) is `Copy`